### ISSUE TITLE
Setting Custom Themes for Tables

### ISSUE THREAD

**Issue by jestarr** [rstudio/gt#386]
Is it be possible to set a custom `gt` theme like you can do using `ggplot2::theme`?  The thought would be to set up consistently used parameters.  For me that would be something to the effect of:

```
df %>% 
  gt() %>% 
  fmt_currency(
    columns = (matches("^salary|^revenue")),
    decimals = 0
  ) %>% 
  fmt_percent(
    columns = starts_with("pct"),
              decimals = 0)
```
I also saw this issue, and I'm not sure if the intent was to include this type of request as a part of it: 
https://github.com/rstudio/gt/issues/238

**Comment by rich-iannone** [comment]
This is a good idea! I think we will explore this a bit in the near future.

**Comment by rich-iannone** [comment]
Fixed in #987
